### PR TITLE
change date format

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

change date format

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/admin/StatusTable.vue [hunk 1]
```diff
@@ -62,8 +62,15 @@ export default {
     formatDate(date) {
       return date ? new Date(date).toLocaleString() : '';
     },
-    getLocalDateFromUtc(date){
-      return date ? new Date(date).toLocaleDateString() : '';
+    formatDateString(date){
+      if(date){
+        var [yyyy, mm, dd] = date.split('-');
```
Comment (loneil) [inline app/frontend/src/components/admin/StatusTable.vue:67]: Can we just print the string as it comes back from the API? Having manual splits isn't maintainable.

Comment (TimCsaky) [reply app/frontend/src/components/admin/StatusTable.vue:67]: ok. it comes back in the format:
'2020-05-22'

Comment (loneil) [reply app/frontend/src/components/admin/StatusTable.vue:67]: If you do want to format it, look at inspectionDateDisplay in InspectionPanel.vue

Comment (loneil) [reply app/frontend/src/components/admin/StatusTable.vue:67]: Yeah either print it as is or use a formatter, no string manipulation, as that will possibly break if a format change from the API ever happens, FE should be able to handle it.

**Pull request merged**